Command/AlignCommand.php

```diff
@@ -55,6 +58,30 @@ protected function configure()
                 InputArgument::REQUIRED,
                 'Environment target name'
             )
+            ->addArgument(
+                'scrollSize',
+                InputArgument::OPTIONAL,
+                'Size of the elasticsearch scroll request',
+                self::DEFAULT_SCROLL_SIZE
+            )
+            ->addArgument(
+                'scrollTimeout',
+                InputArgument::OPTIONAL,
+                'Time to migrate "scrollSize" items i.e. 30s or 2m',
+                self::DEFAULT_SCROLL_TIMEOUT
+            )
+            ->addArgument(
+                'contentType',
+                InputArgument::OPTIONAL,
+                'The content type you wish to align'
+            )
+            ->addOption(
+                'searchQuery',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Query used to find elasticsearch records to import',
+                ''
+            )
```

Comment:
searchQuery must be defined to {} by default otherwise you'll pass an empty string as query